tests/test_expectation_maximization.py

```diff
@@ -254,12 +254,23 @@ def test_insert_slice(test_ir, n_pix):
     )
 
     slices, xyz_rotated_planes = test_ir.generate_slices(
-        map_plane_ones, xy_plane, n_pix, rot_90deg_about_y
+        map_plane_ones, xy_plane, rot_90deg_about_y
     )
 
-    inserted, count = test_ir.insert_slice(slices[0], xyz_rotated_planes[0], n_pix)
-    assert np.allclose(inserted, map_plane_ones)
-    assert np.allclose(count, map_plane_ones)
+    xyz_voxels = test_ir.generate_xyz_voxels(n_pix)
+
+    inserted, count = test_ir.insert_slice(slices[0], xyz_rotated_planes[0], xyz_voxels)
+
+    omit_idx_artefact = 1
+
+    assert np.allclose(
+        inserted[omit_idx_artefact:, omit_idx_artefact:, omit_idx_artefact:],
+        map_plane_ones[omit_idx_artefact:, omit_idx_artefact:, omit_idx_artefact:],
+    )
+    assert np.allclose(
+        count[omit_idx_artefact:, omit_idx_artefact:, omit_idx_artefact:],
+        map_plane_ones[omit_idx_artefact:, omit_idx_artefact:, omit_idx_artefact:],
```

Comment:
in general this test will fail when the rotation is not "on grid". 90 deg puts the counts exactly in the voxel, with a 1-1 correspondence. It's a good test for this 90 deg rotation, but not general.